plugins/woocommerce-admin/client/customize-store/assembler-hub/auto-block-preview.tsx

```diff
@@ -225,18 +210,8 @@ function ScaledBlockPreview( {
 						} );
 					};
 				}, [] ) }
-				aria-hidden
-				tabIndex={ -1 }
-				style={ {
-					width: viewportWidth,
-					height: contentHeight,
-					minHeight:
-						scale !== 0 && scale < 1 && minHeight
-							? minHeight / scale
-							: minHeight,
-				} }
```

Comment:
Removed these style settings because changing the homepage will cause the height to change, but the cannot detect it smoothly. Instead, change the iframe height to 100% and use the iframe's scrollbar (same as site-editor).

Reply:
👍